package.json

```diff
@@ -167,9 +167,9 @@
     "lint-docs-links": "yarn && ./bin/verify-doc-links",
     "lint-yaml": "yamllint **/*.{yaml,yml} --ignore=node_modules/**/*.{yaml,yml}",
     "test": "yarn test-unit && yarn test-timezones && yarn test-cypress",
-    "test-unit": "yarn build-quick:cljs && jest --maxWorkers=2 --config jest.unit.conf.json",
+    "test-unit": "yarn build-quick:cljs && jest --silent --maxWorkers=2 --config jest.unit.conf.json",
     "test-unit-watch": "yarn test-unit --watch",
```

Comment:
If we're going with this approach, I suggest adding it to the watch version too.

Reply:
If you run `yarn test-unit-watch --silent someFilePath`, do you still get warnings?

I spent the last few hours with silent mode on...still getting the warnings ¯\_(¨)_/¯.

Reply:
For me, Jest in watch mode doesn't give console message. There are `UnhandledPromiseRejectionWarning` but they are legitimate problems which need to be sorted out, not a mere console message.